extract-files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright (C) 2010 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DEVICE=seagull
+MANUFACTURER=sony
+STOCK="aosp_d5103-userdebug 5.1.1 LMY48G eng.root.20150718.041511 test-keys"
```

Comment:
Humm. Could it be the reason why my builds are so... Unstable? I modify as soon as I get a Wi-Fi in Rome.mozilla-b2g/device-sony-seagull wrote:In extract-files.sh:
> +#
> +# Licensed under the Apache License, Version 2.0 (the "License");
> +# you may not use this file except in compliance with the License.
> +# You may obtain a copy of the License at
> +#
> +#      http://www.apache.org/licenses/LICENSE-2.0
> +#
> +# Unless required by applicable law or agreed to in writing, software
> +# distributed under the License is distributed on an "AS IS" BASIS,
> +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> +# See the License for the specific language governing permissions and
> +# limitations under the License.
> +
> +DEVICE=seagull
> +MANUFACTURER=sony
> +STOCK="aosp_d5103-userdebug 5.1.1 LMY48G eng.root.20150718.041511 test-keys"

Yes, please. We should not rely on AOSP releases to pull modem firmware, but from Android Stock flashed by Emma for example.

—Reply to this email directly or view it on GitHub.


  
  


Sent Using Firefox OS